installer/Migration/V5_1_0/translation/zz_ZZ.yaml

```diff
@@ -0,0 +1,17 @@
+translations:
```

Comment:
Do we need this language pack? This is used in 4.x I think we can generate translations for `zz_ZZ` using devTool or PHP script